rust-backend/Makefile

```
@@ -0,0 +1,178 @@
+
+# Default target
+.DEFAULT_GOAL := help
+
+# Variables
+BINARY_NAME := weather-server
+DOCKER_IMAGE := weather-backend
+DOCKER_TAG := latest
+
+# Help target
+.PHONY: help
+help: ## Show this help message
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (6).

(maxbodylength)